### PR TITLE
Week 1 Assignments

### PR DESCRIPTION
**Pull request by anugrahpaul2000**
Assignment for Week 1.

### CODE REVIEW

File: Week-1/Assignment-1/readme.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
+<h1>Exercise 1.1 :</h1>
```
Comment (subhadip-pesto) [inline Week-1/Assignment-1/readme.md]: Why you have used html in .dm file?

File: Week-1/Assignment-2/Assignement(1.2).css [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+#detailContainer p i {
+  padding: 2%;
+}
+
+.mainBody table tr:nth-child(even) {
+  background-color: rgb(197, 197, 197);
+  border-spacing: 0;
+}
+.mainBody {
+  padding: 2%;
```
Comment (subhadip-pesto) [inline Week-1/Assignment-2/Assignement(1.2).css]: Dont use % from margins and paddings

File: Week-1/Assignment-2/Assignement(1.2).css [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+}
+.mainBody table {
+  width: 100%;
+  border-spacing: 0;
+}
+.mainBody table tr td {
+  padding: 1%;
+}
+.mainBody table tr td a{
+  color: blue;
```
Comment (subhadip-pesto) [inline Week-1/Assignment-2/Assignement(1.2).css]: try to use hex codes

File: Week-1/Assignment-2/Assignement(1.2).css [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+}
+
+.footer {
+  background-color: rgb(96, 98, 231);
+  height: 10%;
+  padding: 1% 1%;
```
Comment (subhadip-pesto) [inline Week-1/Assignment-2/Assignement(1.2).css]: no percentage.

File: Week-1/Assignment-2/Assignement(1.2).css [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+.mainBody table tr td {
+  padding: 1%;
+}
+.mainBody table tr td a{
+  color: blue;
+}
+
+.footer {
+  background-color: rgb(96, 98, 231);
+  height: 10%;
```
Comment (subhadip-pesto) [inline Week-1/Assignment-2/Assignement(1.2).css]: give fixed height on footer

File: Week-1/Assignment-2/Assignement(1.2).css [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+  padding: 1% 1%;
+  color: white;
+  margin-top: auto;
+  display: flex;
+  flex-direction: row;
```
Comment (subhadip-pesto) [inline Week-1/Assignment-2/Assignement(1.2).css]: not needed

File: Week-1/Assignment-2/Assignement(1.2).css [hunk 7]
```diff
@@ -0,0 +1,128 @@
...
+}
+.playbackControl ul{
+  list-style: none;
+  margin: 0;
+}
+.playbackControl ul li{
+  float: right;
+  padding-left: 5%;
+  padding-right: 5%;
+  font-size: 30;
```
Comment (subhadip-pesto) [inline Week-1/Assignment-2/Assignement(1.2).css]: give unit on font-size

File: Week-1/Assignment-2/Assignement(1.2).css [hunk 8]
```diff
@@ -0,0 +1,128 @@
...
+}ṭ
```
Comment (subhadip-pesto) [inline Week-1/Assignment-2/Assignement(1.2).css]: remove this typo

File: Week-1/Assignment-2/readme.md [hunk 9]
```diff
@@ -0,0 +1,44 @@
+<h1>Exercise 1.2 :</h1>
```
Comment (subhadip-pesto) [inline Week-1/Assignment-2/readme.md]: Why this readme?

File: Week-1/Assignment-1/readme.md (hunk 1 above)
Comment (anugrahpaul2000) [reply Week-1/Assignment-1/readme.md]: To format the view. Just to view in vs code.

File: Week-1/Assignment-2/Assignement(1.2).css (hunk 3 above)
Comment (anugrahpaul2000) [reply Week-1/Assignment-2/Assignement(1.2).css]: Ok sir

File: Week-1/Assignment-2/Assignement(1.2).css (hunk 5 above)
Comment (anugrahpaul2000) [reply Week-1/Assignment-2/Assignement(1.2).css]: Ok sir

File: Week-1/Assignment-1/readme.md (hunk 1 above)
Comment (subhadip-pesto) [reply Week-1/Assignment-1/readme.md]: .md file doesn't know about HTML. please follow this https://medium.com/@saumya.ranjan/how-to-write-a-readme-md-file-markdown-file-20cb7cbcd6f. 

add a readme viewer extension to your vscode.

Comment (anugrahpaul2000) [reply Week-1/Assignment-1/readme.md]: Ok Sir

File: Week-1/Assignment-2/readme.md (hunk 9 above)
Comment (anugrahpaul2000) [reply Week-1/Assignment-2/readme.md]: Readme to Show Question and what to do. For Personal and others Ease.

**Pull request closed without merging**